tests/Types/AsciiStringTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Types;
+
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\AsciiStringType;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class AsciiStringTest extends TestCase
+{
+    /** @var AbstractPlatform|MockObject */
+    private $platform;
+
+    /** @var AsciiStringType */
+    private $type;
+
+    protected function setUp(): void
+    {
+        $this->platform = $this->createMock(AbstractPlatform::class);
```

Comment:
Instead of mocking the platform, I believe the test should use the SQLServer2012Platform that will actually implement the generation of the desired SQL.

Reply:
I think we can actually test the SQL declaration in `SQLServerPlatformTest`. This test then becomes somewhat not needed.

Added this in the latest push if you want to take a look.

Reply:
Unrelated to this do you have any idea why this pipeline step is failing? https://ci.appveyor.com/project/doctrine/dbal/builds/35249676/job/8ytj6a8vv8a8lcv0

It seems like the tests are passing but failing to download some code coverage analysis. Do you have any visibility into whether these failures are related to my changes?

Reply:
It's a known issue, unrelated to your changes.